Wiki Updation #3 (#153)
NilPointer-Software/bdfd-wiki: src/guides/sharecode.md

# Share code

> *"Share code"* is a feature which let's users to share their all bot codes to an another bot.

Reviewer: Should be `that lets users share a copy of their entire bot.`

```suggestion
*"Share code"* is a feature that lets users share a copy of their entire bot.
```

## Creating a share code

> Here is how you can create your own bot share code -

Reviewer: Should be `create a share code of your bot`

```suggestion
Here is how you can create a share code of your bot -
```

- Select your bot in BDFD app homepage.
- Head over to *"Bot Settings"* tab.
- In *"Share code"* section, click *"Create a share code"* button.
- Then, click "Generate share code" button and copy the code.

> Now, you can give the code to your friend or someone and they will have your bot codes in their bot.

Reviewer: Should be `or someone other, and they will be able to create a carbon copy of your bot`

```suggestion
Now, you can give the code to your friend or someone other, and they will be able to create a carbon copy of your bot.
```

> > ⚠️ Once a share code has been used. It gets expired after 1 month.

Reviewer: Should be `has been used, it expires after a month`

```suggestion
> ⚠️ Once a share code has been used, it expires after a month.
```
